django-stubs/contrib/sessions/backends/base.pyi

```diff
@@ -1,29 +1,61 @@
+from _collections_abc import dict_items, dict_keys, dict_values
+from collections.abc import Iterable
 from datetime import datetime, timedelta
-from typing import Any
+from typing import Any, TypeVar, overload
+
+from _typeshed import SupportsKeysAndGetItem
 
 VALID_KEY_CHARS: Any
 
 class CreateError(Exception): ...
 class UpdateError(Exception): ...
 
+_T = TypeVar("_T")
+
 class SessionBase(dict[str, Any]):
     TEST_COOKIE_NAME: str
     TEST_COOKIE_VALUE: str
     accessed: bool
     modified: bool
     serializer: Any
     def __init__(self, session_key: str | None = None) -> None: ...
+    async def aset(self, key: str, value: Any) -> None: ...
     @property
     def key_salt(self) -> str: ...
+    @overload
+    async def aget(self, key: str) -> Any | None: ...
+    @overload
+    async def aget(self, key: str, default: Any) -> Any: ...
+    @overload
+    async def aget(self, key: str, default: _T) -> Any | _T: ...  # type: ignore[misc]
```

Comment:
I would suggest to remove this case, `Any | _T` is still `Any`

Reply:
I copied the signatures over from typeshed, but indeed because `_VT` is `Any` we can remove this overload

Reply:
However, pyright may retain the type information:

![image](https://github.com/user-attachments/assets/8269fdd6-d718-48eb-8b8c-32ed1de4373a)


For a better user experience, maybe this can be useful?

Reply:
I would still prefer not to complicate the signature, cause it does not provide anything extra, but is still visible in the autosuggestions / docs / etc.